File: common/src/main/java/generations/gg/generations/core/generationscore/world/level/block/entities/shrines/WeatherTrioShrineBlockEntity.java
package generations.gg.generations.core.generationscore.world.level.block.entities.shrines;

import generations.gg.generations.core.generationscore.world.level.block.entities.GenerationsBlockEntities;
import net.minecraft.core.BlockPos;
import net.minecraft.world.level.block.state.BlockState;

public class WeatherTrioShrineBlockEntity extends ShrineBlockEntity {
    public WeatherTrioShrineBlockEntity(BlockPos arg2, BlockState arg3) {
        super(GenerationsBlockEntities.WEATHER_TRIO.get(), arg2, arg3);
    }
}

File: common/src/main/java/generations/gg/generations/core/generationscore/common/world/level/block/entities/shrines/WeatherTrioShrineBlockEntity.kt
package generations.gg.generations.core.generationscore.common.world.level.block.entities.shrines

import generations.gg.generations.core.generationscore.common.world.level.block.entities.GenerationsBlockEntities
import net.minecraft.core.BlockPos
import net.minecraft.world.level.block.state.BlockState

class WeatherTrioShrineBlockEntity(pos: BlockPos, state: BlockState) : InteractShrineBlockEntity(GenerationsBlockEntities.WEATHER_TRIO.get(), pos, state)
